src/js/components/DataTable/Resizer.js

```diff
@@ -40,50 +47,61 @@ const Resizer = ({ onResize, property, headerText, messages }) => {
   const [start, setStart] = useState();
   const [width, setWidth] = useState();
   const ref = useRef();
+  const thRef = useRef();
   const { format } = useContext(MessageContext);
 
-  const onMouseDown = useCallback((event) => {
+  useEffect(() => {
     if (ref.current) {
       let element = ref.current;
       // find TH parent
-      while (element && element.nodeName !== 'TH') element = element.parentNode;
+      while (element && element.nodeName !== 'TH') {
+        element = element.parentNode;
+      }
+      thRef.current = element;
+    }
+  }, []);
+
+  const onResizeStart = useCallback((event) => {
+    const clientX = event.touches ? event.touches[0].clientX : event.clientX;
+    if (thRef.current) {
+      const element = thRef.current;
       const rect = element.getBoundingClientRect();
-      setStart(event.clientX);
+      setStart(clientX);
       setWidth(rect.width);
       setActive(true);
     }
   }, []);
 
-  const onMouseMove = useCallback(
+  const onResizeMove = useCallback(
     (event) => {
-      // We determined 12 empirically as being wide enough to hit but
-      // not too wide to cause false hits.
-      const nextWidth = Math.max(12, width + (event.clientX - start));
+      const clientX = event.touches ? event.touches[0].clientX : event.clientX;
+      const nextWidth = Math.max(STEP, width + (clientX - start));
       onResize(property, nextWidth);
     },
     [onResize, property, start, width],
   );
 
-  const onMouseUp = useCallback(() => {
+  const onResizeEnd = useCallback(() => {
     setActive(false);
     setStart(undefined);
     setWidth(undefined);
   }, []);
 
   useEffect(() => {
     const remove = () => {
-      document.removeEventListener('mouseup', onMouseUp);
-      document.removeEventListener('mousemove', onMouseMove);
+      document.removeEventListener('mouseup', onResizeEnd);
+      document.removeEventListener('mousemove', onResizeMove);
     };
 
     if (active) {
-      document.addEventListener('mouseup', onMouseUp);
-      document.addEventListener('mousemove', onMouseMove);
+      document.addEventListener('mouseup', onResizeEnd);
+      document.addEventListener('mousemove', onResizeMove);
```

Comment:
should we also add the following? 
```
  document.addEventListener('touchend', onResizeEnd);
  document.addEventListener('touchmove', onResizeMove);
```

I saw you added down below 
```
          onTouchStart={onResizeStart}
          onTouchMove={start !== undefined ? onResizeMove : undefined}
          onTouchEnd={start !== undefined ? onResizeEnd : undefined}
```

Reply:
See comment below but I think the event listeners might be redundant, will do some testing